pkg/cmd/pr/checkout/checkout_test.go

```diff
@@ -108,32 +93,20 @@ func TestPRCheckout_sameRepo(t *testing.T) {
 	} } } }
 	`))
 
-	ranCommands := [][]string{}
-	//nolint:staticcheck // SA1019 TODO: rewrite to use run.Stub
-	restoreCmd := run.SetPrepareCmd(func(cmd *exec.Cmd) run.Runnable {
-		switch strings.Join(cmd.Args, " ") {
-		case "git show-ref --verify -- refs/heads/feature":
-			return &errorStub{"exit status: 1"}
-		default:
-			ranCommands = append(ranCommands, cmd.Args)
-			return &test.OutputStub{}
-		}
-	})
-	defer restoreCmd()
+	cs, cmdTeardown := run.Stub()
+	defer cmdTeardown(t)
+
+	cs.Register(`git fetch origin \+refs/heads/feature:refs/remotes/origin/feature`, 0, "")
+	cs.Register(`git show-ref --verify -- refs/heads/feature`, 0, "")
+	cs.Register(`git checkout feature`, 0, "")
+	cs.Register(`git merge --ff-only refs/remotes/origin/feature`, 0, "")
 
 	output, err := runCommand(http, nil, "master", `123`)
 	if !assert.NoError(t, err) {
 		return
 	}
 
 	assert.Equal(t, "", output.String())
-	if !assert.Equal(t, 4, len(ranCommands)) {
-		return
-	}
-	assert.Equal(t, "git fetch origin +refs/heads/feature:refs/remotes/origin/feature", strings.Join(ranCommands[0], " "))
-	assert.Equal(t, "git checkout -b feature --no-track origin/feature", strings.Join(ranCommands[1], " "))
-	assert.Equal(t, "git config branch.feature.remote origin", strings.Join(ranCommands[2], " "))
-	assert.Equal(t, "git config branch.feature.merge refs/heads/feature", strings.Join(ranCommands[3], " "))
```

Comment:
These used to be stubbed but are no longer present after the rewrite. Can you confirm that this is no longer needed?

Reply:
No these are needed. This was a mistake on my part of changing test behavior by missing the non-zero exit code. This has been corrected.